bokeh/models/plots.py

```diff
@@ -240,6 +240,23 @@ def add_tile(self, tile_source, **kw):
         self.renderers.append(tile_renderer)
         return tile_renderer
 
+    def add_dynamic_image(self, image_source, **kw):
```

Comment:
I agree. where would be a better place to but this.  Should users simply do:

    p = Plot()
    img_source = ImageSource()
    image_layer = DynamicImageRenderer(img_source)
    p.renderers.append(image_layer)

I kind of like this because is more transparent